ui/jstests/taxonomy/test_add_terms_component.jsx

```diff
@@ -44,21 +72,23 @@ define(['QUnit', 'jquery', 'lodash', 'add_terms_component', 'react',
       // check term title
       var node = React.findDOMNode(component);
       var $vocabTitles = $(node).find(".vocab-title");
-      assert.equal($vocabTitles.length, 1);
-      assert.equal($vocabTitles[0].innerHTML, vocabulary.name);
+      assert.equal($vocabTitles.length, 2);
+      assert.equal($vocabTitles[0].innerHTML, vocabulary2.name);
```

Comment:
Can you add a comment here saying "Assert that the vocabularies are sorted by case insensitive names"